pdf_parser/pdf_parse.py

```diff
@@ -19,6 +19,7 @@ def parse_pdf_document(document):
     cmd = [
             'pdftohtml',
             '-i',
+            '-q',
```

Comment:
Given these flags are fairly opaque, it would be good to add a comment explaining what they mean.

Reply:
👍